src/main/kotlin/com/ampnet/mailservice/service/impl/MailServiceImpl.kt

```diff
@@ -26,7 +26,7 @@ const val FROM_CENTS_TO_EUROS = 100.0
 const val TWO_DECIMAL_FORMAT = "%.2f"
 
 @Service
-@Suppress("TooManyFunctions")
+@Suppress("TooManyFunctions", "SpreadOperator")
```

Comment:
Try not to use suppress. https://arturbosch.github.io/detekt/performance.html#spreadoperator